ports/esp32/Makefile

```diff
@@ -256,8 +256,9 @@ endif
 # these flags are common to C and C++ compilation
 CFLAGS_COMMON = -Os -ffunction-sections -fdata-sections -fstrict-volatile-bitfields \
 	-mlongcalls -nostdlib \
-	-Wall -Werror -Wno-error=unused-function -Wno-error=unused-but-set-variable \
```

Comment:
Ok, now I see in the build log that `-Wfloat-conversion` has warnings in the IDF library, so that's why `-Werror` is removed.  In that case it's better to have it the way you had it before with `-Wfloat-conversion` only applied to our code (not IDF).